File: angular8-springboot-client/src/app/charts/report/energy-efficiency/EnergyConsumptionIndex.ts
import {EnergyCertificate} from '../../../building/enums/energyCertificate';

export class EnergyConsumptionIndex {
  baseline: number;
  thisMonth: number;
  propertiesTarget: number;
  nationalMedian: number;
  baseLineCert: EnergyCertificate;
  thisMonthCert: EnergyCertificate;
  propertyTargetCert: EnergyCertificate;
  nationalMedianCert: EnergyCertificate;
}

File: server/src/main/java/tech/builtrix/web/dtos/bill/EnergyConsumptionIndex.java
package tech.builtrix.web.dtos.bill;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import tech.builtrix.models.building.EnergyCertificate;

/**
 * Created By sahar at 12/9/19
 */

@JsonInclude
@Data
@NoArgsConstructor
@AllArgsConstructor
public class EnergyConsumptionIndex {
	Float baseline;
	Float thisMonth;
	Float propertiesTarget;
	Float nationalMedian;
	EnergyCertificate baseLineCert;
	EnergyCertificate thisMonthCert;
	EnergyCertificate propertyTargetCert;
	EnergyCertificate nationalMedianCert;
}
